validation/promotion_test.go

```diff
@@ -0,0 +1,24 @@
+package validation
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+)
+
+func TestPromotion(t *testing.T) {
+	for _, chain := range OPChains {
+		chain := chain
+		t.Run(perChainTestName(chain), func(t *testing.T) {
+			t.Parallel()
+			if chain.StandardChainCandidate {
+				chain.StandardChainCandidate = false
```

Comment:
What's the purpose of this line? Might be good to add a comment to this function that explains what the function does, which may explain this change here

Reply:
```suggestion
         // promote the chain to standard 
         // by mutating the chainConfig
				chain.StandardChainCandidate = false
```